cvmfs/loader.cc

```diff
@@ -824,7 +843,7 @@ int FuseMain(int argc, char *argv[]) {
 
   // Number of file descriptors
   if (options_manager->GetValue("CVMFS_NFILES", &parameter)) {
-    int retval = SetLimitNoFile(String2Uint64(parameter));
+    int retval = SetLimitNoFile(atoi(parameter.c_str()));
```

Comment:
```suggestion
    int retval = SetLimitNoFile(String2Uint64(parameter));
```
revert unrelated linter changes